nodejs/eks/cluster.ts

```diff
@@ -252,6 +253,7 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         kubeconfig: kubeconfig,
         provider: provider,
         vpcCni: vpcCni,
+        instanceProfileRole: eksRole.role,
```

Comment:
The `eksRole` and `instanceRole` are unrelated, right?  I can’t follow why it’s correct to be passing the former here.

Reply:
Actually - looks like this can just be `instanceRole: instanceRole`?